aim/tests/unit/tools/cli/test_manager.py

```diff
@@ -389,6 +390,51 @@ def test_sync_state_recover(self):
             # Items are updated
             self.assertEqual(4, up.call_count)
 
+    def test_tag_list_command(self):
+        apic_manager_object = mock.Mock()
+
+        def get_mgr():
+            return apic_manager_object
+
+        mo_dict = {'tagInst': {'attributes':
+                   {'dn': 'uni/tn-tn1/tag-openstack_aid'}}}
+        apic_manager_object.apic.list_mo = mock.Mock(
+            return_value=[mo_dict])
+        apic_manager_object.apic.transaction = mock.MagicMock()
+
+        self.mgr.create(self.ctx, resource.Tenant(name='tn1'))
```

Comment:
nit: do we really need to create this tenant for this UT?

Reply:
Yeah - I can remove it. My initial thinking was that we would somehow return that object, but then just decided to mock it. Will remove.